src/main/java/edu/phema/elm_to_omop/translate/criteria/correlation/CorrelatedQuickPair.java

```diff
@@ -0,0 +1,60 @@
+package edu.phema.elm_to_omop.translate.criteria.correlation;
+
+/**
+ * Class representing two QuickResourceAttributePairs, used to represent correlations that we support
+ */
+public class CorrelatedQuickPair {
+    private QuickResourceAttributePair outerResourceAttributePair;
+    private QuickResourceAttributePair innerResourceAttributePair;
+
+    /**
+     * Create from existing QuickResource and QuickResourceAttribute objects
+     *
+     * @param outerQuickResource          The outer QUICK resource
+     * @param outerQuickResourceAttribute The outer QUICK resource attribute
+     * @param innerQuickResource          The outer QUICK resource
```

Comment:
Super minor, but comment should reference "inner" QUICK resource on L15-16, instead of "outer"

Reply:
Thanks, will fix.

Reply:
Actually this class isn't even used. Sorry about that, I'll nix it.